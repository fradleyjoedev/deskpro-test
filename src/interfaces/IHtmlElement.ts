export interface IHTMLElement {
  componentType: string;
  title: string;
  description: string;
}