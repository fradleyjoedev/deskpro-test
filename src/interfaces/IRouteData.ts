export interface IRouteData {
  path: string;
  component: string;
  linkTitle: string;
  pageTitle: string;
  pageDescription: string;
}