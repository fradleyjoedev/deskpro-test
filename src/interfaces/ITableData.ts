export interface ITableData {
  heading: string;
  data: string[];
};