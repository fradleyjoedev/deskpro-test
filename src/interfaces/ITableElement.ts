export interface ITableElement {
  componentType: string;
}