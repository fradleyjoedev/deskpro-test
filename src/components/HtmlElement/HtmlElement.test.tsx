import React from "react";
import { shallow } from 'enzyme';
import { HtmlElement } from './index';
import { IHTMLElement } from "##/interfaces/IHtmlElement";

const data: IHTMLElement = {
  componentType: 'HTMLElement',
  title: 'test',
  description: 'test'
}

describe('Test for HtmlElement', () => {
  it('snapshot', () => {
    const link = shallow(<HtmlElement {...data} />);
    expect(link).toMatchSnapshot();
  });
});