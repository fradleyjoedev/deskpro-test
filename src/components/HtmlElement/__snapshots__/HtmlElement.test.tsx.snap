// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for HtmlElement snapshot 1`] = `
<div
  className="spacing"
>
  <h1>
    test
  </h1>
  <p>
    test
  </p>
</div>
`;
